VAMobile/src/store/slices/appointmentsSlice.ts

```diff
@@ -271,6 +284,9 @@ export const prefetchAppointments =
         } as Params)
       }
 
+      if (getState().appointments.preloadComplete === false && upcomingAppointments?.meta) {
```

Comment:
```suggestion
      if (!getState().appointments.preloadComplete && upcomingAppointments?.meta) {
```
Small syntax sugar suggestion, not critical